sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m

```diff
@@ -328,7 +328,9 @@ - (void)bridgeErrorNotification:(NSNotification *)notification {
 
   NSDictionary *eventBody = @{codeKey:        code,
                               descriptionKey: detail,
-                              userInfoKey:    userInfo};
+                              userInfoKey:    userInfo,
+                              screenTypeKey:  OOStreamPlayer.defaultPlayerInfo.isAudioOnly ?
+                                              @“audio” : @“video”};
```

Comment:
I bet these are wrong quotes, it should be
`" "`

Reply:
Fixed